lib/optimize/RemoveParentModulesPlugin.js

```diff
@@ -58,12 +61,17 @@ class RemoveParentModulesPlugin {
 	apply(compiler) {
 		compiler.plugin("compilation", (compilation) => {
 			compilation.plugin(["optimize-chunks-basic", "optimize-extracted-chunks-basic"], (chunks) => {
-				chunks.forEach((chunk) => {
-					const cache = {};
-					chunk.modules.slice().forEach((module) => {
-						if(chunk.parents.length === 0) return;
-						const dId = "$" + debugIds(module.chunks);
-						let parentChunksWithModule;
+				for(var index = 0, lenChunks = chunks.length; index < lenChunks; index++) {
+					var chunk = chunks[index];
+
+					var cache = {};
```

Comment:
Yes of course. But I think that it is not necessary right now to rewrite a new style.